packages/i18n/web.js

```diff
@@ -0,0 +1,3 @@
+'use strict';
+
+module.exports = require('./dist/src/web');
```

Comment:
Why not avoid additional indirections and not just import everything from `src/types` and also call the file `types`? Basically all of the APIs require file system and currently I don't see where we would need API's for browser environment other than types or constants.